msm/src/precomputed_srs.rs

```diff
@@ -0,0 +1,121 @@
+//! Clone of kimchi/precomputed_srs.rs but for MSM project with BN254
+
+use std::path::PathBuf;
+use std::{fs::File, io::BufReader};
+
+use ark_ff::UniformRand;
+use ark_serialize::{CanonicalDeserialize, CanonicalSerialize, Write};
+
+use kimchi::circuits::domains::EvaluationDomains;
+use poly_commitment::pairing_proof::PairingSRS;
+
+use crate::{BN254G2Affine, Fp, BN254, DOMAIN_SIZE};
+
+/// The path of the serialized BN254 SRS.
+pub fn get_bn254_srs_path() -> PathBuf {
```

Comment:
Why not simply `SRS_FILENAME` which contains the SRS? Like this, we don't have more complicated logic.